variants/diy/hydra/variant.h

```diff
@@ -17,7 +17,9 @@
 #define LED_PIN 2           // add status LED (compatible with core-pcb and DIY targets)
 
 // Radio
-#define USE_SX1262 // E22-900M30S uses SX1262
+//#define USE_SX1262 // E22-900M30S uses SX1262 - 868Mhz
+//#define USE_SX1262 // E22-300M30S uses SX1268 - 433Mhz
+
 #define SX126X_MAX_POWER                                                                                                         \
```

Comment:
Sorry, this change will stop the hydra target precompiled binaries from working. there's no problem defining both chips, as you can see in the t-beam variant.h

Reply:
I defined both modules then, i wasn't sure about it